feat: Refactor Enable And Disable Module Commands (#1918)
kyma-project/cli: docs/gen-docs/kyma_alpha.md

## See also

* [kyma](kyma.md)	 - Controls a Kyma cluster.

> * [kyma alpha add](kyma_alpha_add.md)	 - Enables a resource in the Kyma cluster.

Reviewer: ```suggestion
* [kyma alpha add](kyma_alpha_add.md)	 - Adds a resource to the Kyma cluster.
```
I believe

* [kyma alpha create](kyma_alpha_create.md)	 - Creates resources on the Kyma cluster.

> * [kyma alpha delete](kyma_alpha_delete.md)	 - Disables a resource in the Kyma cluster.

Reviewer: ```suggestion
* [kyma alpha delete](kyma_alpha_delete.md)	 - Deletes a resource from the Kyma cluster.
```
